src/org/zaproxy/zap/view/SessionIncludeInProxyPanel.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ * 
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ * 
+ * Copyright 2010 psiinon@gmail.com
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); 
+ * you may not use this file except in compliance with the License. 
+ * You may obtain a copy of the License at 
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0 
+ *   
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS, 
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+ * See the License for the specific language governing permissions and 
+ * limitations under the License. 
+ * 
+ * Note that this extension ane the other classes in this package are heavily 
+ * based on the orriginal Paros ExtensionSpider! 
+ */
+package org.zaproxy.zap.view;
+
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.model.Model;
+import org.parosproxy.paros.model.Session;
+import org.parosproxy.paros.view.AbstractParamPanel;
+import org.parosproxy.paros.view.View;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class SessionIncludeInProxyPanel extends AbstractParamPanel {
+
+	public static final String PANEL_NAME = Constant.messages.getString("session.proxy.include.title");
+	private static final long serialVersionUID = -8337361808959321380L;
+
+	private JPanel panelSession = null;
+	private MultipleRegexesOptionsPanel regexesPanel;
+
+    public SessionIncludeInProxyPanel() {
+        super();
+ 		initialize();
+	}
+
+	/**
+	 * This method initializes this
+	 */
+	private void initialize() {
+        this.setLayout(new CardLayout());
+        this.setName(PANEL_NAME);
+        regexesPanel = new MultipleRegexesOptionsPanel(View.getSingleton().getSessionDialog());
+        this.add(getPanelSession(), getPanelSession().getName());
+	}
+
+	/**
+	 * This method initializes panelSession
+	 *
+	 * @return javax.swing.JPanel
+	 */
+	private JPanel getPanelSession() {
+		if (panelSession == null) {
+
+			panelSession = new JPanel();
+			panelSession.setLayout(new GridBagLayout());
+			panelSession.setName("Includeproxy");
+
+			GridBagConstraints gridBagConstraints2 = new GridBagConstraints();
+	        GridBagConstraints gridBagConstraints1 = new GridBagConstraints();

```

Comment:
There seems to be some inconsistency in tabs/spaces here vs the previous few lines.